sdf/1.12/mesh_shape.sdf

```diff
@@ -12,6 +12,9 @@
     <element name="max_convex_hulls" type="unsigned int" default="16" required="0">
       <description>Maximum number of convex hulls to decompose into. This sets the maximum number of submeshes that the final decomposed mesh will contain.</description>
     </element>
+    <element name="voxel_resolution" type="unsigned int" default="200000" required="0">
+      <description>Voxel resolution to use for representing the mesh before decomposition. Applicable only to voxel based convex decomposition methods.</description>
```

Comment:
what are the units of the voxel resolution parameter? is it a volumetric density or a number of elements?

Reply:
updated description to say that resolution is the number of voxels. [5416ba5](https://github.com/gazebosim/sdformat/pull/1403/commits/5416ba58eb3b6ab5d6f28bdb07157d9a38fe6be5)